app/splits/[id]/split-info.tsx

```diff
@@ -0,0 +1,18 @@
+import ChampIcon from "@/app/components/champ-icon";
+import { Flex, Heading, Spacer, Text, VStack } from "@chakra-ui/react";
+import champions from "../../_data/champions";
+
+function SplitInfo({ split }: { split: Split }) {
```

Comment:
Personal preference but I feel it's more readable if you deconstruct de param so you don't do `split.item` as much 

```suggestion
function SplitInfo({ { splitChamp, ... } }: { split: Split }) {
```